python/lsst/ip/isr/photodiode.py

```diff
@@ -65,11 +65,11 @@ def __init__(self, timeSamples=None, currentSamples=None, **kwargs):
             if len(timeSamples) != len(currentSamples):
                 raise RuntimeError(f"Inconsitent vector lengths: {len(timeSamples)} vs {len(currentSamples)}")
             else:
-                self.timeSamples = np.array(timeSamples)
-                self.currentSamples = np.array(currentSamples)
+                self.timeSamples = np.array(timeSamples).flatten()
+                self.currentSamples = np.array(currentSamples).flatten()
```

Comment:
These should be `ravel()` not `flatten()` which is a much faster no-copy operation.  (Also below wherever `flatten()` is used).

Reply:
I'm also confused how this could have ever been more than 1d, some explanation would be helpful!